devenv/README.md

```diff
@@ -1,37 +1,39 @@
-This folder contains useful scripts and configuration for...
+# Dev environment setup
 
-* Configuring dev data sources in Grafana
-* Configuring dev & test scenarios dashboards.
-* Creating docker-compose file with DBs and fake data.
+This folder contains useful scripts and configuration so you can:
 
+* Configure dev data sources in Grafana
+* Configure dev and test scenario dashboards.
+* Create docker-compose file with DBs and fake data.
 
-# Dev dashboards and data sources
+
+## Dev dashboards and data sources
 
 ```bash
 ./setup.sh
 ```
 
-After restarting grafana server there should now be a number of data sources named `gdev-<type>` provisioned as well as
-a dashboard folder named `gdev dashboards`. This folder contains dashboard & panel features tests dashboards. 
+After restarting the Grafana server, there should be a number of data sources named `gdev-<type>` provisioned as well as
+a dashboard folder named `gdev dashboards`. This folder contains dashboard and panel features tests dashboards. 
 
-#### Dev dashboards
+## Dev dashboards
```

Comment:
I think this should be under the previous heading, e.g. `### Dashboards`?

Reply:
Actually, that probably would mean that we should remove this heading altogether?

Reply:
Yep